api/services/ocean/spark/schemas/oceanSparkClusterConfigCompute.yaml

```diff
@@ -0,0 +1,16 @@
+type: object
+description: >
+  The object specifying the Ocean Spark Compute configuration.
+properties:
+  useTaints:
+    type: boolean
+    default: true
+    example: false
+    description: |
+      Enable/Disable **ocean-spark** taints on the Ocean Spark VNGs
```

Comment:
```suggestion
      Enable/Disable **ocean-spark** taints on the Ocean Spark VNGs. By default, Spark uses taints to prevent non-Spark pods to run on Ocean Spark VNGs.
```